core/src/test/scala/org/apache/spark/SparkContextSuite.scala

```diff
@@ -1399,6 +1399,14 @@ class SparkContextSuite extends SparkFunSuite with LocalSparkContext with Eventu
     }
     sc.stop()
   }
+
+  test("ShuffleDataIO initialized after application id has been configured") {
```

Comment:
nit. Shall we have a test prefix?

Reply:
Added jira id assuming that is what you meant.